core/domain/topic_domain_test.py

```diff
@@ -821,8 +822,13 @@ def test_update_abbreviated_name(self):
 
     def test_update_thumbnail_filename(self):
         self.assertEqual(self.topic.thumbnail_filename, None)
-        self.topic.update_thumbnail_filename('img.svg')
-        self.assertEqual(self.topic.thumbnail_filename, 'img.svg')
+        with self.assertRaisesRegexp(
+            Exception,
+            'The thumbnail img.svg for topic with id topic_id does not exist'
+            ' in the filesystem.'):
+            self.topic.update_thumbnail_filename('img.svg')
+        self.assertEqual(self.topic.thumbnail_filename, None)
+        self.assertEqual(self.topic.thumbnail_size_in_bytes, None)
```

Comment:
Where's the test for the "success" case?

In general, when writing tests, try to cover all **behaviours** (not just lines of code). This is very important, and if you don't do that proactively, the reviewer will just ask you to do that and it will take more time.

Reply:
Yes, noted. Thanks for the inputs.